front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -53,6 +58,11 @@ export type StdcmConsistProps = {
 const StdcmConsist = ({ isDebugMode, disabled = false }: StdcmConsistProps) => {
   const { t } = useTranslation('stdcm');
 
+  const possibleValueLoadingGauge = ['GA', 'GB'];
```

Comment:
Since it's a hard coded `const`, we can move this outside of the component (between the component and the imports). We also use caps for these consts.
```suggestion
  const GAUGE_LIST = ['GA', 'GB'];
```

Reply:
Done